contracts/Pool.sol

```diff
@@ -71,6 +73,7 @@ contract Pool is PoolFDT {
     event                Cooldown(address indexed lp, uint256 cooldown);
     event      PoolOpenedToPublic(bool isOpen);
     event            PoolAdminSet(address poolAdmin, bool allowed);
+    event      DepositDateUpdated(address indexed lp, uint256 depositDate);
```

Comment:
Moved out of `PoolLib`.

Reply:
You don't need to move out as it should already present in the file, If not then definitely it was an oversight. We need to have at both contracts then only web3 can fetch it when we want to get the logs related to Pool address.

Reply:
Is this a comment or are you suggesting I reinstate the event prototypes I removed from the `PoolLib`?

Reply:
so we are not using the updateDepostDate() function in PoolLib anymore so it is fine to have it in the Pool, My concern was related to moving the event as it should already present in the Pool.sol